#2220: Radcliffe 2: Fix Blog Posts block display list w/images

bobmatyas opened the pull request:
<!-- Thanks for contributing to our free themes! Please provide as much information as possible with your Pull Request by filling out the following - this helps make reviewing much quicker! -->


#### Changes proposed in this Pull Request:

##### Before
<img width="2136" alt="Screenshot on 2022-05-10 at 14-16-25" src="https://user-images.githubusercontent.com/45246438/167696611-f8a20dfe-bcac-4559-8f3b-12d10c3a0f66.png">

##### After

<img width="2129" alt="Screenshot on 2022-05-10 at 14-18-03" src="https://user-images.githubusercontent.com/45246438/167696696-ea39b7de-4064-4b56-af6d-4ca16daa44cd.png">



#### Related issue(s):

#2220

bobmatyas committed:
#2220: Radcliffe 2: Fix Blog Posts block display list w/images

mikachan reviewed (commented):
Thanks for working on this!

This works well for me at resolutions above 600px in width, but at resolutions smaller than this, it looks like we need a similar fix:

![image](https://user-images.githubusercontent.com/1645628/167869296-faeb87dc-27d5-4c6b-940b-9fd6c841e4bf.png)

It looks like the following selectors need a min-width property as well:

![image](https://user-images.githubusercontent.com/1645628/167869474-1eef133c-d6a1-493e-9536-8a692053b09f.png)

![image](https://user-images.githubusercontent.com/1645628/167869544-6458be0d-e322-44f7-9adc-8c42efc4acfd.png)

The `min-width: 50%` fix should probably sit in a media query for min-width 600px, e.g. `@media only screen and (min-width: 600px)`, and then the fix for the smaller resolutions can sit outside of this.

bobmatyas committed:
#2220: Radcliffe 2: Adjust Blog Post Block image display for mobile

bobmatyas commented:
Thanks for the feedback!

I made some adjustments that should fix this on mobile and moved the `min-width: 50%` inside a media query. I also added the additional selectors.

#### Screenshot
<img width="457" alt="Screenshot on 2022-05-11 at 12-21-15" src="https://user-images.githubusercontent.com/45246438/167899348-2b7012e4-babc-4ff4-ad4d-529119a6669d.png">

mikachan reviewed (approved):
Brill, this LGTM! 🚢
